tests/unit/tests_standard_lib/test_requirement_tests/test_test_generator.py

```diff
@@ -106,7 +106,7 @@ def test_extract_params():
             ["requirement.log"],
             [True],
             ["syslog"],
-            {"event": ["<34>Oct 11 22:14:15 machine1 event_1"]},
+            {"event": ["<34>Oct 11 22:14:15 machine1 pr1:event_1"]},
```

Comment:
Basically, our test sample was incorrect, that's why we did not catch this issue?

Reply:
yup true